service/src/main/java/uk/gov/hmcts/reform/fpl/service/PeopleInCaseService.java

```diff
@@ -30,9 +29,9 @@
     private static final String COMMA_DELIMITER = ", ";
 
     public String buildPeopleInCaseLabel(List<Element<Respondent>> respondents,
-                                         Others others) {
-        boolean hasNoOthers = isNull(others) || !others.hasOthers();
-        if (isEmpty(respondents) && hasNoOthers) {
+                                         List<Element<Other>> others) {
+        boolean hasNoActiveOthers = others.isEmpty();
```

Comment:
could use the same check as the one below in the if

Reply:
although then you would have to call the method twice so this is more "efficient" (note that the efficiencies are insignificant here)